src/Cevou/Behat/ScreenshotCompareExtension/ServiceContainer/ScreenshotCompareExtension.php

```diff
@@ -81,55 +83,66 @@ public function configure(ArrayNodeDefinition $builder)
 
         $builder
             ->children()
-                ->arrayNode('sessions')
-                    ->useAttributeAsKey('name')
-                    ->prototype('array')
-                    ->children()
-                        ->scalarNode('adapter')->defaultValue('default')->end()
-                        ->arrayNode('crop')
-                            ->children()
-                                ->integerNode('left')->defaultValue(0)->min(0)->end()
-                                ->integerNode('right')->defaultValue(0)->min(0)->end()
-                                ->integerNode('top')->defaultValue(0)->min(0)->end()
-                                ->integerNode('bottom')->defaultValue(0)->min(0)->end();
+            ->arrayNode('screenshot_config')
```

Comment:
I like the indentation that was here before better. You can see the blocks much easier.